conf/messages

```diff
@@ -276,7 +276,7 @@ eligibility.check.span.para=To apply for Marriage Allowance, you must be married
 eligibility.check.married=Does this apply to you?
 eligibility.check.married.legend=Are you married or in a legally registered civil partnership?
 eligibility.check.yes = Yes
-eligibility.check.married.error=You are not eligible for Marriage Allowance because you are not married or in a legally registered civil partnership.
+eligibility.check.married.error=You are not eligible for Marriage Allowance because you are not married or in a legally registered civil partnership.</br>You can make a Marriage Allowance claim if your partner has since died, provided the conditions are met. It can be backdated up to 4 years, as with current claims. If this applies to you call HMRC on 0300 200 3300.
```

Comment:
could you not separate this into two separate message keys? as it is not good to have HTML in the message file where possible

Reply:
If you try something like:
        <div class="tamc-js-hidden error-inline validation-summary" id="married-js-error">
            @Html(Messages("eligibility.check.married.error1"))
            @Html(Messages("eligibility.check.married.error2"))
        </div>
It wont create new like, using <p>  or <div> will show extra lining space which is also we dont want so simple solution here is use BR